src/sso/sso.ts

```diff
@@ -12,16 +14,33 @@ export class SSO {
     projectID,
     redirectURI,
     state,
-  }: AuthorizeURLOptions): string {
+  }: AuthorizationURLOptions): string {
     const { apiHostname = API_HOSTNAME } = this.workos.options;
 
     const query = queryString.stringify({
       domain,
       client_id: projectID,
       redirect_uri: redirectURI,
+      response_type: 'code',
```

Comment:
Just to make sure I'm understanding this correctly: the `response_type` value informs the queried auth server which type of grant (in this case, an auth code) to return in its response, correct?

Reply:
yup!

Reply:
Great, thanks! 😄